### PR TITLE
convert screens/internal to typescript

### PR DESCRIPTION
**Pull request by schamp**

### CODE REVIEW

File: src/screens/internal/AllStates/AllStates.tsx [hunk 1]
```diff
@@ -3,22 +3,26 @@ import { LoadingScreen } from './AllStates.style';
...
-function State({ state }) {
+function State({ state }: { state: StateType }) {
   const projections = useProjectionsFromRegion(state);
 
   // Projections haven't loaded yet
   if (!projections) {
     return <LoadingScreen></LoadingScreen>;
   }
-  const stateName = projections.stateName;
+  const stateName = state.name;
```
Comment (schamp) [inline src/screens/internal/AllStates/AllStates.tsx:25]: projections didn't have stateName, but we're passing a state in, so it seems reasonable to use the state name from that, if we have found a projectino.

File: src/screens/internal/AllStates/AllStates.tsx [hunk 2]
```diff
@@ -3,22 +3,26 @@ import { LoadingScreen } from './AllStates.style';
...
 import { MetricChart } from 'components/Charts';
-import regions from 'common/regions';
+import regions, { State as StateType } from 'common/regions';
 
 function AllStates() {
-  return regions.states.map(state => {
-    return <State key={state.stateCode} state={state} />;
-  });
+  return (
+    <>
```
Comment (schamp) [inline src/screens/internal/AllStates/AllStates.tsx:10]: the context in which AllStates() was used required a single child, not a list, so I wrapped in a <>

File: src/screens/internal/AllStates/AllStates.tsx (hunk 1 above)
Comment (schamp) [reply src/screens/internal/AllStates/AllStates.tsx:25]: this actually fixes a bug

**Review by pnavarrc — APPROVED** [review]
Looks good!

**Pull request merged**